### PR TITLE
Ports Hippie nooses

### PR DESCRIPTION
**Pull request by FlattestGuitar**
Lynching on a whole new level!

Fun for the whole family!

Why wouldn't you want this?!

I ported the exact functionality of Hippie's noose. I'm not 100% on whether this should behave exactly this way, but I couldn't find a better option. Open to suggestions!

- You can only construct these while standing on a table/chair/bed/toilet/stool

- You don't need to take the table out from under people in order for them to start choking. As soon as the're buckled they start dying. (so you wouldn't have to unwrench chairs from under someone to execute them, a dirty solution, but I think it's the best one)

- No neck-breaking, only slow oxygen damage kills you.

- Obviously, people who don't have to breathe are unaffected by nooses.

- You can free yourself from this if you don't have cuffs, but with cuffs on it will take as long to free yourself as with a chair - way too long.

:cl: FlattestPancake
add: Ports nooses from old Hippie
/:cl:

### CODE REVIEW

**Comment by TullyBurnalot** [comment]
Wot

File: code/modules/power/cable.dm [hunk 1]
```diff
@@ -514,6 +511,169 @@ var/global/list/datum/stack_recipe/cable_coil_recipes = list(
...
+
+/obj/structure/stool/bed/noose/attackby(obj/item/W, mob/user)
+	if(istype(W, /obj/item/weapon/wirecutters))
+		user.visible_message("[user] cuts the noose.", "<span class='notice'>You cut the noose.</span>")
+		if(buckled_mob && buckled_mob.mob_has_gravity())
+			buckled_mob.visible_message("<span class='danger'>[buckled_mob] falls over and hits the ground!</span>",\
+										"<span class='userdanger'>You fall over and hit the ground!</span>")
+			buckled_mob.adjustBruteLoss(10)
+		var/obj/item/stack/cable_coil/C = new(get_turf(src))
+		C.amount = 15
```
Comment (Fox-McCloud) [inline code/modules/power/cable.dm]: Pretty sure you can specify this in `new`

File: code/modules/power/cable.dm [hunk 2]
```diff
@@ -514,6 +511,169 @@ var/global/list/datum/stack_recipe/cable_coil_recipes = list(
...
+		qdel(src)
+		return
+	..()
```
Comment (Fox-McCloud) [inline code/modules/power/cable.dm]:
```
else
	return ..()
```

File: code/modules/power/cable.dm [hunk 3]
```diff
@@ -514,6 +511,169 @@ var/global/list/datum/stack_recipe/cable_coil_recipes = list(
...
+	var/obj/item/organ/head = H.get_organ("head")
+	if(head == null || istype(head, /obj/item/organ/external/stump))
+		to_chat(user, "<span class='warning'>They have no head!</span>")
+		return FALSE
+
+	add_fingerprint(user)
+
+	if(H == user && buckle_mob(H))
+		H.visible_message(\
+			"<span class='suicide'>[H] ties \the [src] over their neck!</span>",\
```
Comment (SamHPurp) [inline code/modules/power/cable.dm]: iirc `\the` is now not supposed to be used?

File: code/modules/power/cable.dm [hunk 4]
```diff
@@ -514,6 +511,169 @@ var/global/list/datum/stack_recipe/cable_coil_recipes = list(
...
+			"<span class='suicide'>You tie \the [src] over your neck!</span>")
```
Comment (SamHPurp) [inline code/modules/power/cable.dm]: here also

File: code/modules/power/cable.dm [hunk 5]
```diff
@@ -514,6 +511,169 @@ var/global/list/datum/stack_recipe/cable_coil_recipes = list(
...
+				if(M && M.buckled)
+					to_chat(M, "<span class='warning'>You fail to untie yourself!</span>")
+				return
+			if(!M.buckled)
+				return
+			M.visible_message(\
+				"<span class='warning'>[M] unties the noose over their neck!</span>",\
+				"<span class='notice'>You untie the noose over your neck!</span>")
+			M.Weaken(3)
+		if(istype(M, /mob/living/carbon/human))
```
Comment (Fox-McCloud) [inline code/modules/power/cable.dm]: helperssss

**Comment by taukausanake** [comment]
I don't like how you start choking right away once buckled. Table flipping is a thing. You can pick up stools, maybe kicking them could be a solution? Same for chairs maybe? Not sure about the others though; bed and toilet won't move.

Can you break necks normally? We don't even have neck organs if I recall. If we did I can see it causing brute and oxy-loxx

File: code/modules/power/cable.dm (hunk 4 above)
Comment (SamHPurp) [inline code/modules/power/cable.dm]: and further with all the other visible messages lol

File: code/modules/mob/living/carbon/human/human_defines.dm [hunk 6]
```diff
@@ -73,3 +73,5 @@ var/global/default_martial_art = new/datum/martial_art
 
 	var/datum/body_accessory/body_accessory = null
 	var/tail // Name of tail image in species effects icon file.
+
+	var/noosed = FALSE //for nooses
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/human_defines.dm]: This is kinda gross.

File: code/modules/power/cable.dm [hunk 7]
```diff
@@ -514,6 +511,169 @@ var/global/list/datum/stack_recipe/cable_coil_recipes = list(
...
+
+/obj/structure/stool/bed/noose/user_buckle_mob(mob/living/carbon/human/H, mob/user)
+	if(!in_range(user, src) || user.incapacitated() || !ishuman(H))
+		return FALSE
+
+	if(get_turf(H) != get_turf(src))
+		return FALSE //Can only noose someone if they're on the same tile as noose
+
+	var/obj/item/organ/head = H.get_organ("head")
+	if(head == null || istype(head, /obj/item/organ/external/stump))
```
Comment (Fox-McCloud) [inline code/modules/power/cable.dm]: !head

**Comment by SamHPurp** [comment]
Can we just remove the sodding suicide verb now?

**Comment by Rumiluntti** [comment]
Everyone knows there is no ceiling.
Emulsions=ruined

File: code/modules/mob/living/carbon/human/human_defines.dm (hunk 6 above)
Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/human_defines.dm]: This is ~~kinda~~ really ~~gross~~ awful. FTFY

File: code/modules/power/cable.dm [hunk 8]
```diff
@@ -514,6 +511,169 @@ var/global/list/datum/stack_recipe/cable_coil_recipes = list(
...
+	over.layer = FLY_LAYER
+
+/obj/structure/stool/bed/noose/Destroy()
+	processing_objects.Remove(src)
+	return ..()
+
+/obj/structure/stool/bed/noose/post_buckle_mob(mob/living/M)
+	if(buckled_mob)
+		overlays.Add(over)
+		src.layer = MOB_LAYER
```
Comment (tigercat2000) [inline code/modules/power/cable.dm]: No src.

File: code/modules/power/cable.dm [hunk 9]
```diff
@@ -514,6 +511,169 @@ var/global/list/datum/stack_recipe/cable_coil_recipes = list(
...
+		processing_objects.Add(src)
+		M.dir = SOUTH
+		animate(M, pixel_y = initial(pixel_y) + 8, time = 8, easing = LINEAR_EASING)
```
Comment (tigercat2000) [inline code/modules/power/cable.dm]: `pixel_y = M.get_standard_pixel_y_offset() + 8`

**Comment by Anticept** [comment]
Typically, nooses cut off blood supply to the brain. So those that don't breathe may still be affected if they have a circulatory system with jugulars that can be pinched.

**Comment by Regen1** [comment]
This is in extremely poor taste.

There's literally nothing to this PR other than a meme taken from hippie, and it has the added side effect of being an extremely real situation for people.

What if my mum hung herself? Or your Brother was hanging from his neck in his bedroom. Every other suicide method is some silly thing that doesn't really reflect to a real life situation, but this does.

It's adding an entire new element to the game for the sole purpose of portraying a method of suicide which can deeply affect people. I personally think that this has no place on paradise, and we're better off without it.

PS: No I've not had any firsthand experience with this, but I'm trying to be considerate for people who might've

**Comment by EvadableMoxie** [comment]
I just... I feel this crosses a line I don't think we should cross.  I don't want paradise to be.... this. Whatever this is. It's hard to really put into words because it's purely an emotional response but I just think this goes a bit too far.

**Comment by Necaladun** [comment]
Most of the suicide verbs we have are pretty silly, and I'd prefer to keep suicide that way.

I don't want to encourage lynching, especially with how many memes there are around lynching people based on species lines. This is very often referencing lynching of blacks, which is a kinda atmosphere I don't want to encourage.

**Comment by tigercat2000** [comment]
After conferring between the Heads of Staff and Maintainers, it was decided to close this PR, as it is not seen as something that befits the atmosphere of Paradise Station and would only serve to cause more problems than any boons.

**Pull request closed without merging**